js/eventListener.js

```diff
@@ -6,26 +6,26 @@ let guling = document.querySelectorAll(".pots");
 
 //let keys = document.querySelectorAll(".key");
 
-let playlist = ["", "./assets/gulingtanga-notes/Db4.wav", "./assets/gulingtanga-notes/Eb4.wav", "", "./assets/gulingtanga-notes/Gb4.wav", "./assets/gulingtanga-notes/Ab4.wav", "./assets/gulingtanga-notes/Bb4.wav", "./assets/gulingtanga-notes/C4", "./assets/gulingtanga-notes/D4", "./assets/gulingtanga-notes/E4", "./assets/gulingtanga-notes/F4", "./assets/gulingtanga-notes/G4", "./assets/gulingtanga-notes/A5", "./assets/gulingtanga-notes/B5", "./assets/gulingtanga-notes/C5"];
+let notes=document.querySelectorAll(".notes");
 
 //gulingtangan onclick
 
 guling.forEach(function(gulingtangan){
     gulingtangan.addEventListener("click", function(){
-      console.log(gulingtangan);
+      gulingtangan.children[1].play();
     })
 })
 
 //gulingtangan keypress
 
 document.addEventListener("keypress", logKey);
 
-let keyArray = ["", "1", "2", "", "7", "8", "9", "q", "w", "e", "r", "u", "i", "o", "p"];
+let keyArray = ["", "1", "2", "", "7", "8", "9", "", "q", "w", "e", "r", "u", "i", "o", "p"];
 
 function logKey(e){
     for(i=0; i<keyArray.length; i++){
         if(e.key === keyArray[i]){
-        console.log(playlist[i]);
```

Comment:
Also, make a new function called playNote(note) for which note is like "C4". And the function will create a new sound with the note C4. Then the function is called when it is clicked by mouse or pressed by the right keyboard key_